src/Client.ts

```diff
@@ -15,6 +15,10 @@ export interface ClientOptions extends Eris.ClientOptions {
 	prefix: string;
 	/** If true, prefix matching is case-sensitive. */
 	caseSensitivePrefix?: boolean;
+	
+	// Whether or not, the command can be ran using case-sensivity. For example: !ping and !Ping
```

Comment:
Please use the other types of comments here
```suggestion
	/** Whether or not, the command can be ran using case-sensivity. For example: !ping and !Ping */
```